packages/console/src/components/Tip/Tooltip/index.module.scss

```diff
@@ -3,3 +3,7 @@
 .content {
   @include _.multi-line-ellipsis(6);
 }
+
+.anchor {
+  display: inline-block;
+}
```

Comment:
Why do we need this? Any issue you encountered?

Reply:
The anchor wapper is a `div` and will stretch to fulfill its parent's width.

Reply:
1. This might be a big assumption here.  Since Tooltip looks like a commonly used component. With this style update, it forces the child component as an inline-block element, regardless of the original element type. 
2. Centralize the tooltip should not rely on the anchor element's type. Either a full-width div element or an inline element should work as expected. 

@charIeszhao Do you mind following up on this?

Reply:
Remove this assumption, code updated.

Reply:
Well, this is actually the right move, after a quick discussion we realized it is something missing in the existing component.
The container wrapper should tightly wrap the inner element, and thus the inline property suits that well

Reply:
Per offline discussion, use inline-block is expected.